### ISSUE TITLE
Add channel for rubocop 1.60

### ISSUE THREAD

**Issue by hampld** [codeclimate/codeclimate-rubocop#364]
Hi,

Rubocop has released a new version: https://github.com/rubocop/rubocop/releases/tag/v1.60.0

Please add a channel that mirrors that release.

Thanks.

*tumbleweed rolling through the issues*

**Comment by rebeccafae** [comment]
I second this request!

**Comment by rebeccafae** [comment]
The beta channel is moving really fast, and I still need a channel that uses 1.60

I was able to use 1-56-3 for a while, but now new updates are making that not possible. Can we please get ANY newer version made into a permanent channel (maybe 1.63.5 https://github.com/codeclimate/codeclimate-rubocop/issues/370)? I know a lot of us need this!

Taking it from any commit on the beta channel would work just fine.

@fede-moya  tagging you since you helped merge some of those beta channel PRs

**Comment by fede-moya** [comment]
Hi @rebeccafae ,

Apologies for the delay on this matter. I'm going to push to get this done as soon as possible. I'll keep you posted on the progress.

By the way, does RuboCop version `1.63.5` work?

**Comment by rebeccafae** [comment]
> Hi @rebeccafae ,
> 
> Apologies for the delay on this matter. I'm going to push to get this done as soon as possible. I'll keep you posted on the progress.
> 
> By the way, does RuboCop version `1.63.5` work?

Yes. I will be thrilled with any newer locked version. Thank you so much!

**Comment by hampld** [comment]
actually, as rubocop is already on `1.64.1`, that would be great ...

**Comment by hampld** [comment]
I see you merged `1.64.1` but there is still no channel for any of the more current rubocop versions. any chance we can get that before summer vacations?

**Comment by hampld** [comment]
Obsolete and never even started.
